### PR TITLE
Continue upload ui

### PR DESCRIPTION
**Pull request by AaronVasquez**
Depends on https://github.com/oysterprotocol/oyster-streamable/pull/37

### CODE REVIEW

File: package.json.orig [hunk 1]
```diff
@@ -0,0 +1,173 @@
+{
```
Comment (EdmundMai) [inline package.json.orig]: what is this file for?

File: src/components/shared/progress-bar.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  border-radius: 6px;
+  background-color: transparent;
+  height: 48px;
+  max-width: 450px;
+`;
+
+const ProgressBar = ({ progress }) => (
+  <ProgressBorder className="progress">
+    <div
+      class="progress-bar"
```
Comment (EdmundMai) [inline src/components/shared/progress-bar.jsx]: styled components

File: src/components/upload-started/upload-started-slide.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+const UploadStartedSlide = ({ chunksProgress }) => {
+  const description =
+    chunksProgress >= 99.999 // epsilon b/c float comparison.
+      ? "Confirming upload..."
+      : "File is being broken into chunks and each chunk encrypted...";
+
+  return (
+    <Slide title="Upload Started" image={ICON_UPLOAD}>
+      <p className="transaction-confirmed-instructions">
```
Comment (EdmundMai) [inline src/components/upload-started/upload-started-slide.tsx]: styled components

File: src/components/upload-started/upload-started-slide.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        {description}
+        <Spinner isActive={chunksProgress === 0} className="download-spinner" />
+      </p>
+      <p className="transaction-confirmed-instructions">
```
Comment (EdmundMai) [inline src/components/upload-started/upload-started-slide.tsx]: same

File: src/components/shared/progress-bar.jsx (hunk 2 above)
Comment (AaronVasquez) [reply src/components/shared/progress-bar.jsx]: This is from twitter bootstrap

File: src/components/upload-started/upload-started-slide.tsx (hunk 3 above)
Comment (AaronVasquez) [reply src/components/upload-started/upload-started-slide.tsx]: I'm just reusing this class. Can we do the conversion in a separate PR, this one is already pretty big

**Pull request merged**